MAINTAINERS

```diff
@@ -0,0 +1 @@
+Juanjo Alvarez <juanjo@sourced.tech>
```

Comment:
please include ` (@juanjux)` at the end or in other cases generator will write a link with your email instead of github profile